quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java

```diff
@@ -74,7 +74,7 @@
      * Android application UI elements
      */
     private TextView videoStatusTextView;
-    private CameraCapturer cameraCapturer;
+    private CameraCapturerCompat cameraCapturer;
```

Comment:
Would prefer to be more explicit here so `CameraCapturerCompat cameraCapturerCompat` it's more obvious we are using this class in other parts of the code.